app/util/SanitizeInput.scala

```diff
@@ -6,13 +6,24 @@ import scala.util.matching.Regex
 
 object SanitizeInput {
   val sanitizeRegex: Regex = "(<.*?>|<[^>]*$)".r
+  val hiddenPriorities = List("edition", "training")
 
   def fromString(s: String) = sanitizeRegex.replaceAllIn(s, "")
 
-  def fromConfigSeo(config: ConfigJson): ConfigJson = {
-    config.copy(fronts = config.fronts.mapValues(sanitizeSeoInputFromFront))
+  def sanitizeConfig(config: ConfigJson): ConfigJson = {
+    runSanitize(config, sanitizeSeoInputFromFront, ensureHiddenPriorities)
```

Comment:
why have we added `ensureHiddenPriorities` function here?

Reply:
I've just read your implementation notes! I'm not sure if we need this extra check as we've made the editions work just like training fronts which have stayed well hidden for at least as long as I've been here. Was there some specific use case you wanted to guard against?

Reply:
The edge case was that while moving quickly we might forget to add this on the client, and as a result I decided to just add in `training` as there seemed no reason not to. It can be removed, it just seemed safer while we were developing.

Reply:
If we are not going to be developing on these as our actual editions might as well remove this.